Library/Homebrew/test/utils/svn_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/svn"
+
+describe Utils do
+  describe "#self.svn_available?" do
+    before(:each) do
+      if described_class.instance_variable_defined?(:@svn)
```

Comment:
Perhaps adding a new method `clear_svn_location_cache` or similar and call that in the `before`.